frappe/website/utils.py

```diff
@@ -549,7 +549,7 @@ def add_preload_headers(response):
 	try:
 		preload = []
 		strainer = SoupStrainer(re.compile("script|link"))
-		soup = BeautifulSoup(response.data, "lxml", parse_only=strainer)
+		soup = BeautifulSoup(response.data, "html.parser", parse_only=strainer)
```

Comment:
AFAIK lxml is significantly faster 😅 

We might wanna do some perf bench before this change. 


(Also this use of BS4 is hacky here 😅)

Reply:
Nope, nothing particular right now... will look into it 👍

Reply:
@ankush since we are using SoupStrainer (to selectively parse)... changing parser should not have noticeable performance impact.